taskcluster/ci/signing/kind.yml

```diff
@@ -42,3 +38,12 @@ job-template:
                 nightly: BNs
                 default: Bs
         kind: build
+    worker-type:
```

Comment:
I just reordered this template alphabetically. I started to the same in other mobile projects about a month ago.